### ISSUE TITLE
tensor.softmax() not support

### ISSUE THREAD

**Issue by williamlzw** [dotnet/TorchSharp#914]
https://github.com/ultralytics/ultralytics/blob/main/ultralytics/nn/modules.py
line80
self.conv(x.view(b, 4, self.c1, a).transpose(2, 1).softmax(1)).view(b, 4, a)

softmax(1)? not support in torchsharp

**Comment by NiklasGustafsson** [comment]
`torch.softmax()` is undocumented, so it's not been included in TorchSharp:

https://pytorch.org/docs/stable/search.html?q=torch.softmax&check_keywords=yes&area=default

However, it was reported recently in a bug, and a fix will appear in the next release. In the meantime, use `torch.special.softmax()`
